modules.d/99base/dracut-lib.sh

```diff
@@ -1125,6 +1125,9 @@ remove_hostonly_files() {
             [ -e "$line" ] || [ -h "$line" ] || continue
             rm -f "$line"
         done < /lib/dracut/hostonly-files
+        if [ -n "$DRACUT_SYSTEMD" ]; then
```

Comment:
Would not be modules.d/98dracut-systemd/dracut-cmdline.sh better place to do this work ? That would make the code more modular and you would not need to check if DRACUT_SYSTEMD ?

Reply:
Yes, that makes sense. I added the check in dracut-lib just because other functions do the same.